_articles/de/legal.md

```diff
@@ -113,6 +113,7 @@ Durch das Hinzufügen von "Papierkram", den einige für unnötig, schwer verstä
 Einige Situationen, in denen Sie eine zusätzliche Kontributionsvereinbarung für Ihr Projekt in Betracht ziehen sollten, sind z.B:
 
 * Ihre Anwälte wollen, dass alle Mitwirkenden ausdrücklich die Kontributionsbedingungen akzeptieren (_unterschreiben_, on- oder off-line), vielleicht weil sie der Meinung sind, dass die Open-Source-Lizenz selbst nicht ausreicht (obwohl sie ausreicht!). Wenn dies die einzige Sorge ist, sollte eine Kontributionsvereinbarung, welche die Open-Source-Lizenz des Projekts bestätigt, ausreichen. Das [jQuery Individual Contributor License Agreement](https://contribute.jquery.org/CLA/) ist ein gutes Beispiel für eine leichtgewichtige, zusätzliche Kontributionsvereinbarung. Für manche Projekte kann ein [Developer Certificate of Origin](https://github.com/probot/dco) eine Alternative sein.
+* Sie oder Ihre Anwälte wollen Entwickler\*innen bestätigen lassen, dass Ihre commits autorisiert sind. Viele Projekte nutzen dafür das [Developer Certificate of Origin](https://developercertificate.org/). Die Node.js-Community bspw. [nutzt ein DCO](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md) anstatt ihres [vorherigen CLAs](https://nodejs.org/en/blog/uncategorized/notes-from-the-road/#easier-contribution). [DCO Probot](https://github.com/probot/dco) bietet eine simple Möglichkeit, in Ihrem Projekt ein solches DCO automatisch einzufordern.
```

Comment:
I would write **Commits** with an uppercase c.
I would say **Zum Beispiel nutzt die Node.js-Community ein DCO ...**
I would change **simple** to **einfache**